learnify/mobile/lib/features/home/components/course_preview.dart

```diff
@@ -0,0 +1,62 @@
+part of '../view/home_screen.dart';
+
+class _CoursePreview extends StatelessWidget {
+  const _CoursePreview({
+    required this.textKey,
+    required this.participantNumber,
+    // ignore: unused_element
```

Comment:
I guess this ignore is unnecessary now.

Reply:
Since it is called without an image parameter in the home ( see home_screen.dart line 21) this ignore is required. It can be removed while implementing the row or when the backend connection is complete. Until then it should be ignored so as not to get a warning.